cmd/moby/trust_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
the standard library is unfortunately not very helpful for making sense of boolean/error checks unless we want to wrap in conditionals (I'm not aware of a better way?)

An array would work but I think that's unmaintainable since this is already 29 cases and there will surely be more in the future. Did you have a certain format in mind?  I also anticipate that we can use this for other tests - I can port to `config_test.go` in a followup fairly easily